.jenkins/Jenkinsfile

```diff
@@ -18,27 +19,38 @@ podCreator.initMain(label: label) {
                 testsNames = testsNamesRaw.values().toList()
                 println(testsNames)
             }
-            stage('Creating tests and building docker files') {
-                def scriptToUse = libraryResource 'generate_tests.groovy'
-                for (int i = 0; i < testsNames.size(); i++) {
-                    def dirPath = "${testsNames[i]}".toString()
-                    def jobName = "${testsNames[i]}".toString().replace("/","-")
-                    def dockerTag = TestUtilities.getDockerTagNameForRegression(testFolder, jobName)
-                    withCredentials([
-                            string(credentialsId: 'ROOKOUT_REGGRESION_TOKEN', variable: 'ROOKOUT_TOKEN'),
-                            string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
-                            string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'AWS_SECRET_ACCESS_KEY')
-                    ]) {
-                        if(Utilities.checkIfFileExist(this, dirPath, "Dockerfile.test")){
-                            sh("cd ${dirPath} && docker build -f Dockerfile.test -t ${dockerTag} . && gcloud docker -- push ${dockerTag}")
-                        }
-                        else{
-                            sh("cd ${dirPath} && docker build . -t ${dockerTag} && gcloud docker -- push ${dockerTag}")
-                        }
+
+            def scriptToUse = libraryResource 'generate_tests.groovy'
+            for (int i = 0; i < testsNames.size(); i++) {
+                def dirPath = "${testsNames[i]}".toString()
+                echo "dirPath: ${dirPath}"
+                def jobName = "${testsNames[i]}".toString().replace("/","-")
+                withCredentials([
+                        string(credentialsId: 'ROOKOUT_REGGRESION_TOKEN', variable: 'ROOKOUT_TOKEN'),
+                        string(credentialsId: 'AWS_ACCESS_KEY_ID', variable: 'AWS_ACCESS_KEY_ID'),
+                        string(credentialsId: 'AWS_SECRET_ACCESS_KEY', variable: 'AWS_SECRET_ACCESS_KEY')
+                ]) {
+                    if(Utilities.checkIfFileExist(this, dirPath, "Dockerfile.test"))
+                    {
+                        dockerfile = "Dockerfile.test"
+                    }
+                    else
+                    {
+                        dockerfile = "Dockerfile"
                     }
-                    TestUtilities.createTest(this, scriptToUse, jobName, pwd() + "/${testsNames[i]}", "Jenkinsfile", testFolder)
+                    podCreator.initKaniko(branch: env.GIT_BRANCH,
```

Comment:
Won't we a have a problem with the slash replacement in branches with slashes?

Reply:
We didn't remove the line, it's just went down

Reply:
We will, on it